Sources/TokamakDemo/TransitionDemo.swift

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 7/13/21.
+//
+
+import TokamakShim
+
+private struct ColorOverlayModifier: ViewModifier {
+  var color: Color
+
+  func body(content: Content) -> some View {
+    content
+      .background(color)
+  }
+}
+
+struct TransitionDemo: View {
+  @State private var isVisible = false
+  @State private var isInnerVisible = false
+
+  var body: some View {
+    VStack {
+      Button(isVisible ? "Hide" : "Show") {
+        withAnimation(.easeInOut(duration: 3)) { isVisible.toggle() }
```

Comment:
If you remove the animation from the outer section (so just `isVisible.toggle()`), then click hide while the inner section is animating, the outer section will animate as well.

Reply:
This should be resolve now.